src/components/ComboBox/ComboBox.stories.tsx

```diff
@@ -50,17 +50,28 @@ const defaultItems = [
     label: 'アイテムのラベルが長い場合（ダミーテキストダミーテキストダミーテキストダミーテキスト）',
     value: 'value-6',
   },
+  {
+    label: (
+      <>
+        <div>
+          アイテムのラベルが<span>ReactNodeの場合</span>
+        </div>
+        <p>（ダミーテキストダミーテキストダミーテキストダミーテキスト）</p>
```

Comment:
細かいですが、button には基本的にフレージング・コンテンツしか入らないため、div や p は入りません。
縦に並べたい場合は Stack[as=span] など別の方法を使う必要があります。

```suggestion
        <span>
          アイテムのラベルが<span>ReactNodeの場合</span>
        </span>
        <span>（ダミーテキストダミーテキストダミーテキストダミーテキスト）</span>
```

Reply:
全くもってその通りなので、修正させていただきました 🙏 
https://github.com/kufu/smarthr-ui/pull/3270/commits/a376a825c81c7c9a95fb0b32f9d87deff0ce15fb